docs/src/modules/components/AppNavDrawerItem.js

```diff
@@ -127,9 +127,11 @@ const ItemButton = styled(Item, {
       }
       return theme.palette.text.primary;
     })(),
-    fontSize: theme.typography.pxToRem(depth === 0 ? 14.5 : 12),
+    fontSize: theme.typography.pxToRem(depth === 0 ? 14.5 : 11),
+    textTransform: depth === 0 ? 'none' : 'uppercase',
+    letterSpacing: depth === 0 ? 'normal' : '.1rem',
```

Comment:
I'm not sure about "normal", are we overriding the default value? Would this work?

```suggestion
    letterSpacing: depth === 0 ? null : '.1rem',
```

Reply:
I don't really know what's the best approach 😅  @siriwatknp wanna weigh in?

Reply:
`null` should work.